pipeline/interpolate_gene.py

```diff
@@ -194,12 +205,17 @@ def main(
 
     output_dir = Path(output_dir)
     output_dir.mkdir(parents=True, exist_ok=True)
+    output_path = str(output_dir / f"{experiment_id}-{interpolator_name}-interpolated-{image_type}")
 
-    np.save(
-        output_dir
-        / f"{experiment_id}-{interpolator_name}-interpolated-{image_type}.npy",
-        predicted_volume,
-    )
+    if saving_format == "npy":
+        np.save(
+            output_path + ".npy",
+            predicted_volume,
+        )
+    else:
+        from convert_npy_nrrd import HEADER
+        HEADER["sizes"] = np.array(predicted_volume.shape)
```

Comment:
So the `sizes` is not hardcoded but the `space directions` are? Once we work on the "custom resolution" feature this might be an issue. No?

IMO we either hardcode both of them or none of them.

Reply:
Good points! Addressed here: [d53c77d](https://github.com/BlueBrain/Deep-Atlas/pull/64/commits/d53c77d999e553cb479e5a8e14f1dd7786ba4954)

Reply:
What about the `space directions` though? They seem to dictate the number of microns "per pixel", right?

Reply:
I am not sure `space directions` has be changed. For me, `dimension` has to change depending if it is a grayscale or RGB volume. But `space dimension` and `space directions` have to stay the same. No ?

Reply:
As nothing is implemented for the resolution, I added a TO DO in the resolution PR (https://github.com/BlueBrain/Deep-Atlas/pull/48#issue-1276951358)